docs/getting-started-on-linux.md

```diff
@@ -0,0 +1,109 @@
+## Getting Started
```

Comment:
>Getting [](http://example.com/codeflow?start=3&length=7)

Delete this file. I believe this is the README.md file in the public repo.

Reply:
It might make sense to add details for building on windows/mac here as well as opposed to the main readme.
The only thing is there is some msdoc link pointing at our main readme, which we'll need to update if so.

I'll delete this for now and I'll open a task for better documentation.